### PR TITLE
architecture adjustments, thanks vscode

### PR DESCRIPTION
**Pull request by bennycio**
Changes:

- **Changed** almost everything but did not actually change much. Just moved things around and renamed them with the handy dandy refactoring capabilities of vscode

Considerations:

- A lot of things are renamed and moved around, if anyone has any opinions on where things should be instead of where they are or what they should be named, I am all ears!

### CODE REVIEW

File: persist/collection.go [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+	"context"
+	"time"
+
+	"github.com/mikeydub/go-gallery/runtime"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+const collectionColName = "glry_collections"
```
Comment (fbrcsftnr) [inline persist/collection.go:13]: should we update the collection names as well?

File: persist/mongo.go [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package glry_core
+package persist
```
Comment (fbrcsftnr) [inline persist/mongo.go:1]: noticed that files within glry_db/ and glry_core/ are given `package persist`. is this intentional? should we rename the directory to be `persist/`?

Comment (fbrcsftnr) [reply persist/mongo.go:1]: ^ nevermind!! i see that it's been moved haha

**Review by fbrcsftnr — APPROVED** [review]

File: persist/collection.go (hunk 1 above)
Comment (bennycio) [reply persist/collection.go:13]: Very true, I will adjust those as well! Thank you for pointing this out to me

**Pull request merged**